src/renderers/shaders/ShaderChunk/lights_physical_fragment.glsl.js

```diff
@@ -116,5 +116,30 @@ material.roughness = min( material.roughness, 1.0 );
 
 	#endif
 
+#endif
+
+#ifdef USE_ANISOTROPY
+
+	#ifdef USE_ANISOTROPYMAP
+
+		mat2 anisotropyMap = mat2( anisotropyVector.x, anisotropyVector.y, - anisotropyVector.y, anisotropyVector.x );
+		vec2 anisotropyV = anisotropyMap * ( 2.0 * texture2D( anisotropyMap, vUv ).rg - vec2( 1.0 ) );
+
+	#else
+
+		vec2 anisotropyV = anisotropyVector;
+
+	#endif
+
+	material.anisotropy = length( anisotropyV );
+	anisotropyV /= material.anisotropy;
+	material.anisotropy = saturate( material.anisotropy );
+
+	// Roughness along the anisotropy tangent is the material roughness, while the bitangent roughness increases with anisotropy.
+	material.alphaB = pow2( mix( material.roughness, 1.0, pow2( material.anisotropy ) ) );
```

Comment:
During this week's PBR TSG it was noted that all other anisotropy systems we studied (Disney, MaterialX, Adobe ASM, Google Filament, Dassault Enterprise PBR) all agree that the bitangent roughness is lower and tangent roughness is higher, so setting alphaB to a greater value here is opposite the convention.  Sadly, there is no consistency among any of these systems as to what the exact formula is, and indeed this PR introduces a new anisotropic strength formula here.  I've attempted to [chart the different strength formulas](https://docs.google.com/presentation/d/1EMjwJOrSy5XfR3Vzug_M0w4CRhTcslawwXiYo1yOEb0/edit#slide=id.g2125a0cb896_0_0).